### PR TITLE
kola/test/docker: enable `docker.selinux` for all arch

### PR DESCRIPTION
**Pull request by tormath1**
SELinux is now supported on ARM64

Signed-off-by: Mathieu Tortuyaux <mtortuyaux@microsoft.com>

## Testing done

```
# get image from SELinux PR https://github.com/kinvolk/coreos-overlay/pull/1245
wget https://storage.googleapis.com/flatcar-jenkins/developer/developer/boards/arm64-usr/2021.09.01%2Bdev-flatcar-master-3470/flatcar_production_image.bin.bz2
sudo ./bin/kola run --channel alpha --board arm64-usr --key ${HOME}/.ssh/id_rsa.pub -k -b cl --qemu-bios ./flatcar_production_qemu_uefi_efi_code.fd  -p qemu --qemu-image ./flatcar_production_image.bin docker.selinux
```

### CODE REVIEW

File: kola/tests/docker/docker.go [hunk 1]
```diff
@@ -55,12 +55,11 @@ type simplifiedDockerInfo struct {
...
-		ClusterSize:   1,
-		Name:          "docker.selinux",
-		Distros:       []string{"cl"},
-		Channels:      []string{"alpha", "beta"},
-		Architectures: []string{"amd64"},
+		Run:         dockerSELinux,
+		ClusterSize: 1,
+		Name:        "docker.selinux",
+		Distros:     []string{"cl"},
+		Channels:    []string{"alpha", "beta"},
```
Comment (pothos) [inline kola/tests/docker/docker.go:62]: While at it we should switch this to using `MinVersion` or we will forget this when the next channel transition happens

Comment (pothos) [reply kola/tests/docker/docker.go:62]: `MinVersion:       semver.Version{Major: 2942},` should mean everything newer, including the current beta?

Comment (tormath1) [reply kola/tests/docker/docker.go:62]: why not `MinVersion: semver.Version{Major: 2984},` ? SELinux changes will be available only in the next `alpha` :thinking:

Comment (pothos) [reply kola/tests/docker/docker.go:62]: Ah yes, I just thought about the existing `beta` value

Comment (tormath1) [reply kola/tests/docker/docker.go:62]: This is an edge case actually:
  * SELinux `amd64` is available on `alpha`/`beta`
  * SELinux `arm64` is only on `main` atm

so:
* `MinVersion: semver.Version{Major: 2942},`: will make the test fail on `arm64` for both beta / alpha
* `MinVersion: semver.Version{Major: 2984},`: will make the test disappears for `amd64` for both beta / alpha

Comment (pothos) [reply kola/tests/docker/docker.go:62]: Yes, `MinVersion: semver.Version{Major: 2984},` is the right one :D

Comment (pothos) [reply kola/tests/docker/docker.go:62]: Better to skip the tests than have them fail when doing a bugfix release

Comment (pothos) [reply kola/tests/docker/docker.go:62]: ah, wait, didn't get it fully when reading the first time… we have no way to specify the version per architecture, complicated. I think it's ok to let it fail for arm64 on Alpha bugfix releases then and use `2942`.

Comment (tormath1) [reply kola/tests/docker/docker.go]: ok done with `2942` :)

**Review by pothos — APPROVED** [review]
